### ISSUE TITLE
Forgetting/reset tags

### ISSUE THREAD

**Issue by Macter4** [berwyn/muzei-ponies#7]
The application in the latest version, after some time, forgets the tags and starts using the default values. The manually entered tags still are located in the appropriate text box. It looks as if the application downloaded a few graphics when entering tags, and later it could not build the right query. The case is not individual, because my colleague has the same problem and we observed him a week ago.

**Comment by berwyn** [comment]
Yeah, I've noticed this myself. It's on the slate for the next beta build but I've been bogged down at work, unfortunately, and haven't had time to fix it.

If you want to take a look though, I do accept PRs!

**Comment by Macter4** [comment]
Sorry, but I can't help you. I'm sysadmin and the only code I create is horrible spaghetti to automate my work ;)
